src/anemometry.jl

```diff
@@ -219,14 +219,15 @@ Two different models are used depending on the Mach number:
 
 - [1] Ward, D. T. (1993). Introduction to flight test engineering. Elsevier
     Science Ltd. (page 12)
+- [2] Fundamentals of Aerdynamics, 5th edition, J.D.Anderson Jr (page 550)
 """
 function compressible_qinf(M, p)
 
     if M < 1
         pt = p * (1 + (γ - 1) / 2 * M*M) ^ (γ / (γ - 1))
-    else
-        # TODO: implementation for M > 1
-        error("Not Implemented yet")
+    else    # (M > 1)
```

Comment:
Great! can we get a test for this? https://github.com/parthp08/FlightMechanics.jl/blob/master/test/anemometry.jl#L111

Reply:
will add tests in next PR